Cubical/Experiments/Combinatorics.agda

```diff
@@ -1,109 +0,0 @@
-{-
```

Comment:
Maybe keep? Can you make it so that it gets checked when checking Experiments.Everything?

Reply:
This one does not check.

Reply:
Commented out broken code, added to Everything